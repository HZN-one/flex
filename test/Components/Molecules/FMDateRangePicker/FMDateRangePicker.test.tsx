/**
 * @jest-environment jsdom
 */

import * as React from "react";
import "@testing-library/jest-dom";
import { render } from "@testing-library/react";

import { FMDateRangePicker } from "@Molecules";

describe("FMCard", () => {
  it("should match snapshot", () => {
    const { container } = render(
      <FMDateRangePicker
        testID="date-range-picker-test"
        label="Pick Date Range"
        value={[
          new Date("Wed Mar 16 2022 00:00:00 GMT+0700"),
          new Date("Wed Mar 16 2022 00:00:00 GMT+0700"),
        ]}
        setValue={value => {
          console.log(value);
        }}
      />
    );
    expect(container).toMatchSnapshot();
  });
});
