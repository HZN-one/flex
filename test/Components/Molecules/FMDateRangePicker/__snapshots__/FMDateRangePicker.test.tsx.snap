// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FMCard should match snapshot 1`] = `
<div>
  <div
    class="css-17vdz66"
  >
    <div
      class="MuiBox-root css-0"
    >
      <input
        placeholder="mm/dd/yyyy"
        type="hidden"
        value=""
      />
      <input
        placeholder="mm/dd/yyyy"
        type="hidden"
        value=""
      />
      <div
        class="MuiFormControl-root MuiFormControl-fullWidth MuiTextField-root css-wb57ya-MuiFormControl-root-MuiTextField-root"
        data-testid="input-date-range-picker"
      >
        <label
          class="MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated MuiInputLabel-shrink MuiInputLabel-outlined MuiFormLabel-root MuiFormLabel-colorPrimary MuiFormLabel-filled css-1sumxir-MuiFormLabel-root-MuiInputLabel-root"
          data-shrink="true"
          for="mui-1"
          id="mui-1-label"
        >
          Pick Date Range
        </label>
        <div
          class="MuiOutlinedInput-root MuiInputBase-root MuiInputBase-colorPrimary MuiInputBase-fullWidth MuiInputBase-formControl MuiInputBase-adornedEnd css-154xyx0-MuiInputBase-root-MuiOutlinedInput-root"
        >
          <input
            aria-invalid="false"
            class="MuiOutlinedInput-input MuiInputBase-input MuiInputBase-inputAdornedEnd css-nxo287-MuiInputBase-input-MuiOutlinedInput-input"
            id="mui-1"
            readonly=""
            type="text"
            value="16 Mar - 16 Mar"
          />
          <div
            class="MuiInputAdornment-root MuiInputAdornment-positionStart MuiInputAdornment-outlined MuiInputAdornment-sizeMedium css-ittuaa-MuiInputAdornment-root"
          >
            <span
              class="notranslate"
            >
              â€‹
            </span>
            <button
              class="MuiButtonBase-root MuiIconButton-root MuiIconButton-sizeMedium css-78trlr-MuiButtonBase-root-MuiIconButton-root"
              data-testid="icon-button-date-range"
              tabindex="0"
              type="button"
            >
              <span
                aria-hidden="true"
                class="material-icons notranslate MuiIcon-root MuiIcon-fontSizeMedium css-1a42aau-MuiIcon-root"
                data-testid="icon-date-range"
              >
                date_range
              </span>
              <span
                class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
              />
            </button>
          </div>
          <fieldset
            aria-hidden="true"
            class="MuiOutlinedInput-notchedOutline css-1d3z3hw-MuiOutlinedInput-notchedOutline"
          >
            <legend
              class="css-7nxzgd"
            >
              <span>
                Pick Date Range
              </span>
            </legend>
          </fieldset>
        </div>
      </div>
    </div>
  </div>
</div>
`;
